lib/manager/components/validation/MobilityDataValidationResult.js

```diff
@@ -0,0 +1,149 @@
+              )
+            })}
+          </tr>
+        ))}
+      </tbody>
+    </Table>
+  )
+}
+
+// eslint-disable-next-line complexity
+const renderNoticeDetail = (notice) => {
+  switch (notice.code) {
+    case 'too_many_rows':
+      notice.csvRowNumber = notice.rowNumber
+    case 'fare_transfer_rule_duration_limit_type_without_duration_limit':
+    case 'fare_transfer_rule_duration_limit_without_type':
+    case 'fare_transfer_rule_missing_transfer_count':
+    case 'fare_transfer_rule_with_forbidden_transfer_count':
+      notice.filename = 'fare_transfer_rules.txt'
+    case 'empty_file':
+    case 'emtpy_row':
+    case 'missing_timepoint_column':
+    case 'missing_required_file':
+    case 'missing_recommended_file':
+    case 'unknown_file':
+      return (
+        <ul style={{marginTop: '-10px'}}>
+          {notice.sampleNotices.map((notice) => (
+            <li>
+              {notice.filename}
+              {notice.csvRowNumber && `: row ${notice.csvRowNumber}`}
+            </li>
+          ))}
+        </ul>
+      )
+    default:
+      return (
+        <NoticeTable notices={notice.sampleNotices} />
+      )
+  }
+}
+
+const MobilityDataValidationResult = (props) => {
+  const { notice } = props
+  const rule = rules.find((rd) => rd.rule === notice.code)
+  if (!rule) return null
+
+  const errorClass = `gtfs-error-${mobilityDataValidationErrorMapping[notice.severity]}`
+  const [expanded, setExpanded] = useState(notice.totalNotices < 2)
+
+  const onRowSelect = () => setExpanded(!expanded)
+
+  return (
+    <ListGroupItem style={{ overflow: 'scroll', padding: 0 }}>
+      <div style={{ padding: '10px 15px' }}>
+        <h4
+          onClick={onRowSelect}
+          onKeyDown={onRowSelect}
+          style={{ cursor: 'pointer' }}
+        >
+          <span
+            className={`buffer-icon ${errorClass}`}
+            title={`${toSentenceCase(notice.severity)} priority`}
+          >
+            <Icon
+              type={validationErrorIconLookup[mobilityDataValidationErrorMapping[notice.severity]]}
+            />
+          </span>
+          {toSentenceCase(notice.code.replaceAll('_', ' ').toLowerCase())}
+          <span className={errorClass}>
+            {' '}
+            &mdash; {notice.totalNotices} case
+            {notice.totalNotices > 1 ? 's' : ''} found
+          </span>
+          <span className={`pull-right`}>
+            <Icon type={expanded ? 'caret-up' : 'caret-down'} />
+          </span>
+        </h4>
+        {expanded && (
+          <>
+            <p><Markdown>{rule.description}</Markdown></p>
+            <p>
+              <a
+                href={`https://github.com/MobilityData/gtfs-validator/blob/master/RULES.md#${notice.code}`}
+                target='_blank'
+                rel='noreferrer'
+              >
+                More details
+              </a>
+            </p>
+          </>
+        )}
+      </div>
+      {expanded && renderNoticeDetail(notice)}
```

Comment:
Notices with a lot of sample issues create huge tables here, which are very cumbersome to scroll through. We should implement the `Load More` functionality that the current validator has for large results: 
![image](https://user-images.githubusercontent.com/63798641/224373118-29d7df51-ba13-45cf-ac64-6e167b95dcc5.png)
vs.
![image](https://user-images.githubusercontent.com/63798641/224373196-5956f733-36fc-4953-ab70-c4d530184326.png)

Reply:
I should add the load more for the DT validation results is only there because of how large each validation row is data-size wise

Reply:
I was coming at this from the angle that for a table with scrolling, you can't scroll horizontally without reaching the bottom of the table, which is 1000s of entries long. Maybe the scroll should be sticky?

Reply:
I'm not necessarily married to the load more functionality, but it might be a bit weird if the tables operate in different ways for large samples.

Reply:
I think if anything the load more functionality should be removed from the regular validator. It's a confusing experience and I don't think a very useful one (can't search etc)

what do you mean you can't scroll horizontal without reaching the bottom? is the scrolling not working for you?